vue-components/src/components/Button.vue

```diff
@@ -19,20 +26,50 @@ import Vue from 'vue';
  * https://bugzilla.mozilla.org/show_bug.cgi?id=1581369#c5
  */
 /* eslint-enable no-trailing-spaces */
-export default Vue.extend( {
+export default defineComponent( {
 	name: 'Button',
 	props: {
 		/**
 		 * The type of the button
+		 *
+		 * @values neutral, progressive, destructive
 		 */
 		type: {
 			type: String,
 			validator( value: string ): boolean {
-				return [ 'neutral', 'primaryProgressive', 'primaryDestructive' ].includes( value );
+				return [ 'neutral', 'progressive', 'destructive' ].includes( value );
 			},
 			default: 'neutral',
 		},
+		/**
+		 * @values normal, primary
+		 */
+		variant: {
+			type: String,
+			validator( value: string ): boolean {
+				return [ 'normal', 'primary' ].includes( value );
```

Comment:
Should it include quiet too?

Reply:
not yet, I think that will be added in your PR

Reply:
but eventually yes :)

Reply:
ah I see.